muffnn/mlp/tests/test_mlp_classifier.py

```diff
@@ -65,11 +68,31 @@ def check_multilabel_predictions(clf, X, y):
     assert np.sum(np.abs(probabilities.sum(axis=1) - 1)) > 0.01,\
         "multilabel classifier is outputting distributions"
 
+def check_multilabel_predictions_na(clf, X, y):
+    predicted = clf.fit(X, y).predict(X)
+    #change nans to zero to 'mask' those values
+    is_na=(y==-1)
+    predicted[is_na]=0
+    y[is_na]=0
+
+    assert_equal(predicted.shape, y.shape)
+    predicted[is_na]=0
+    y[is_na]=0
+    assert_array_equal(predicted, y)
+
+    probabilities = clf.predict_proba(X)
+    assert_equal(probabilities.shape, y.shape)
+    assert_array_equal((probabilities >= 0.5).astype(np.int), predicted)
```

Comment:
Do you need to use the is_nan mask here on probabilities after converting to int?